graphql/version_resolver.go

```diff
@@ -69,9 +69,6 @@ func (r *versionResolver) BuildVariants(ctx context.Context, obj *restModel.APIV
 		obj.Activated = version.Activated
 	}
 
-	if !utility.FromBoolPtr(obj.Activated) {
```

Comment:
I don't think this is the right approach. This is an optimization to avoid over fetching tasks when there shouldn't be any.

The reason the tasks are failing is because the activated flag for the version is being set to false at some point even though there are tasks.